src/platforms/common/profiling/index.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Set Up
+sidebar_order: 5000
+supported:
+  - android
+  - apple
+notSupported:
+  - unity
+  - dart
+  - flutter
+  - rust
+  - native
+  - php
+  - ruby
+  - javascript
+  - javascript.vue
+  - react-native
+  - dotnet
+  - python
+  - java
+  - java.spring-boot
+  - java.spring
+  - go
+  - javascript.cordova
+  - native.breakpad
+  - native.crashpad
+  - native.minidumps
+  - unreal
+  - native.wasm
+description: "Learn how to enable profiling in your app if it is not already set up."
+---
+
+<Note>
+
+Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. For access, contact us at [profiling@sentry.io](mailto:profiling@sentry.io).
+
+</Note>
+
+<Note>
+
+If you don't have access to Profiling please reach out to [profiling@sentry.io](mailto:profiling@sentry.io).
+
+</Note>
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments.
+
+## Enable Performance Monitoring
+
+Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK:
+
+<PlatformSection supported={["apple"]}>
+
+```swift
+SentrySDK.start { options in
+    options.dsn = "..."
+    options.tracesSampleRate = 1.0
+}
+```
+
+</PlatformSection>
+
+<PlatformSection supported={["android"]}>
+
+In `AndroidManifest.xml`:
```

Comment:
The user is already on an Android page, and we have two sentences before the code snippet ending in a colon. We don't do this on the Apple version of these docs. Can we remove this or do we need this information here?
```suggestion
```